docs/docs/events/index.md

```diff
@@ -0,0 +1,71 @@
+# Events
+
+Events refer to the activity in a system that tracee monitors. There are two main types of events, built-in events that are part of Tracee and custom events, which references the way users define other events that Tracee should monitor. 
+
+As part of built-in events, there are six types of events:
+
+* syscalls 
+* network 
+* security 
+* lsm 
+* containers 
+* misc
+
+This section documents all of the different events that Tracee exposes.
+
+## Defining events
```

Comment:
Defining events sounds like adding new events to Tracee. maybe change to "selecting events"